andela/travel_tool_front
# #160371227 Update notification pane

#### What does this PR do?
 Updates the notification pane to use the message accompanying a notification

#### Description of Task to be completed?
- update notification pane to use the message from the database
- change avatar before `add a comment` text on request modal to that of the logged in user

#### How should this be manually tested?
- Clone this repo - git clone https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Install dependencies and start the server - make start
- Create a new travel request or click on a request after routing to My Approvals page and approve the request if any in other to have notification/s
Click on the Bell icon on the navbar to see the new notification

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160371227](https://www.pivotaltracker.com/story/show/160371227)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
